crates/viewer/re_space_view_map/src/visualizers/mod.rs

```diff
@@ -73,6 +77,21 @@ impl GeoSpan {
     }
 }
 
+/// Extend a span to include another span, if any.
+pub fn update_span(span: &mut Option<GeoSpan>, other: Option<GeoSpan>) {
```

Comment:
`GeoSpan` is like a bounding box on a map, no?

How does that work at ±180° longitude, i.e. how do you make a `GeoSpan` that includes the pacific ocean?

Reply:
We should maybe have a TODO for this them, i.e. finding the minimal bounding box for an input like:

```
longitude: -179     lat = 10
longitude: +179    lat = -10   
```

Reply:
I opened an issue listing some of the work required for a decent handling at the boundaries: https://github.com/rerun-io/rerun/issues/7993   🐇🕳️